src/songuploader/settings/base.py

```diff
@@ -164,3 +164,5 @@
     ("de", _("German")),
     ("en", _("English")),
 ]
+
+MEDIA_ROOT = BASE_DIR / "media"
```

Comment:
please seperate into prod and dev settings:
`MEDIA_ROOT = BASE_DIR / "media" into dev`
`MEDIA_ROOT = "/var/www/data/media" into prod`